src/Processors/QtiV2/In/SharedPassageMapper.php

```diff
@@ -1,20 +1,22 @@
 <?php
-
+include "../../../Utils/General/DOMHelper.php";
```

Comment:
Why is this here? We should always be using the autoloader in here, we should never need to make includes. If this is related to the issue with the mismatched namespace and directory path on https://github.com/Learnosity/learnosity-qti/pull/55/files#diff-2cff45284c03e635f2e75e341a6c781eR3, I'd fix that instead.

Reply:
Done